css/style.css

```diff
@@ -1,32 +1,41 @@
-/**
- * Beware that Qt Assistant (QTextBrowser) supports only a limited set of 
- * HTML/CSS: http://qt-project.org/doc/qt-5/richtext-html-subset.html
- */
```

Comment:
I didn't test it yet but I don't see anything which looks like a problem when it is ignored. Otherwise I would have commented that line.